server/server.js

```diff
@@ -0,0 +1,99 @@
+const express = require('express');
+const app = express();
+const cors = require("cors")
+
+app.use(express.json())
+app.use(cors())
+
+app.listen(3000, function () {
+    console.log("Server is running");
+});
+
+const something = {
+	name: "My Name",
+	city: "My City",
+    message: "Hi"
+}
+
+let todos = [
+    { 
+        title: 'Do laundry', 
+        done: false, 
+        id: 1
+    },
+    { 
+        title: 'Make dinner', 
+        done: true, 
+        id: 2
+    },
+];
+
+app.get("/", function (req, res) {
+    res.send(something);
+});
+
+app.get("/todos", function (req, res) {
+    res.send(todos);
+});
+
+let currentId = todos.length + 1;
```

Comment:
hi, this is a bit wrong to have `currentId` as global variable.
as per task:
```
Refactor `POST /task` to return to the client the whole new task object as response.
```

so, in post() method you add a new todo.
that's the place where you can get the real id:
```js
const receivedTodo = req.body;
todos.push(receivedTodos);
receivedTodo.id = todos.length - 1;
res.status(201).send(receivedTodo);
```